lib/polish_geeks/dev_tools/config_manager.rb

```diff
@@ -0,0 +1,53 @@
+module PolishGeeks
+  module DevTools
+    # Config manager for commands
+    class ConfigManager
+      def initialize(file_name)
+        @file_name = file_name
+      end
+
+      # @return [Boolean] if there is a config present
+      def present?
+        application? || local?
+      end
+
+      # @return [String] path to config file
+      def path
+        application? ? application_path : local_path
+      end
+
+      private
+
+      # @return [Boolean] if there is a config present locally
+      def local?
+        !@file_name.nil? && !local_path.nil?
+      end
+
+      # @return [String] path to local config file
+      def local_path
+        fetch_path(DevTools.gem_root)
+      end
+
+      # @return [Boolean] if there is a config present in the application
+      def application?
+        !@file_name.nil? && !application_path.nil?
```

Comment:
Nope, we want to return boolean, changing that to what you propose will end in returning nil or path.